code/modules/vehicle/sportscar.dm

```diff
@@ -0,0 +1,47 @@
+/obj/vehicle/car
+	name = "sports car"
+	desc = "A very luxurious vehicle."
+	icon = 'icons/vehicles/sportscar.dmi'
+	icon_state = "sportscar"
+	generic_pixel_x = 0
+	generic_pixel_y = 4
+	vehicle_move_delay = 1
+	var/static/image/carcover = null
```

Comment:
it's semi-synonymous with ```/global```
it also has the quirk that you can access ```/static``` vars from a typepath WITHOUT using ```initial()```

ie: the usual
```
var/obj/item/sword/I = /obj/item/sword
world << initial(I.attackpower)
```
would be
```
var/obj/item/sword/I = /obj/item/sword
world << I.attackpower
```

But yes the main usage here was so that only one version of the image ever needs to exist for all atvs.